index.html

```diff
@@ -15,35 +15,28 @@
   <body>
     <div class="main-content">
       <header id="top">
-        <div id="nav_bar">
+        <nav>
           <h1 class="logo"><a href="#top">CED<sup>music</sup></a></h1>
-          <nav>
             <ul>
-              
               <li><a href="#au">About Us</a></li>
               <li><a href="#serv">Our Services</a></li>
               <li><a href="#team_meet">Meet the Team</a></li>
               <li><a href="#contact">Contact Us</a></li>
             </ul>
           </nav>
-        </div>
       </header>
       <main>
         <banner>
           <div class="title_container">
-            <div>
-              <h1 id="title">CED Music</h1>
-            </div>
-            <div>
-              <span id="caption">The place for all your music, production, and sound system needs.</span>
-            </div>
+            <h1 id="title">CED Music</h1>
+            <p id="caption">The place for all your music, production, and sound system needs.</p>
           </div>
         </banner>
         <!--About Us-->
         <section>
           <div class="about_us" id="au">
             <div class="main_au">
-              <h1 id="">About Us</h1>
+              <h2 id="">About Us</h2>
```

Comment:
So this isn't an 'h1' because it is sitting 'under' the h1 at the top of the page. Generally headings should live in a hierarchy such that directly under h1s there are only h2s, directly under h2s there are only h3s etc.

Again, this is about the semantics of HTML, you can use CSS to make the headings look however you like!